ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -845,6 +913,9 @@ func validateRayJobSpec(rayJob *rayv1.RayJob) error {
 	if rayJob.Spec.BackoffLimit != nil && *rayJob.Spec.BackoffLimit < 0 {
```

Comment:
Add a validation to check if `DELETE_RAYJOB_CR_AFTER_JOB_FINISHES` env and `DeletionPolicy` are both set, return an error to ask users to use `DeletionPolicy` only.

Reply:
seems not necessary